src/test/scala/loamstream/googlecloud/GoogleCloudChunkRunnerTest.scala

```diff
@@ -504,7 +504,7 @@ final class GoogleCloudChunkRunnerTest extends FunSuite with ProvidesEnvAndResou
       assert(client.deleteClusterInvocations() === 0)
       assert(client.isClusterRunningInvocations() === 3)
       
-      client.deleteCluster()
+      client.stopCluster()
```

Comment:
Just because this code is different, I want to make sure the terminology is good? On AWS `stop != terminate/delete`, it just means relinquishing the CPU time back to AWS, but the VM is still there in a usable state.

Is stopping different from deletion or the same for google cloud? If different (like AWS), why the change here?

Reply:
This is a fair point.  I went with "stop" because that's what `hailctl` calls that operation (even though it boils down to `gcloud ... delete`), and I originally planned to use `hailctl` for everything.  `stop` is also a dual of `start`.

`delete` seemed too tied to Google's terminology, but if that's what AWS calls it, then that's a good reason to use `delete`.

>Is stopping different from deletion or the same for google cloud?

`hailctl`'s "stop" is the same as `gcloud`'s "delete", which means shut down and get rid of the cluster and all the nodes in it.

Reply:
To be clear, in AWS it's called "terminate", but it's essentially "delete", "nuke", "obliterate", and otherwise "remove from existence". ;-)

Reply:
I'm going to keep this as `stop` just because it's the dual of `start`.  If/when we add AWS support, I'll switch this to a name that makes sense for GCP and AWS.